mypyc/ir/ops.py

```diff
@@ -1412,6 +1412,55 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_load_mem(self)
 
 
+class SetMem(Op):
+    """Write a memory location.
+
+    *(type *)src = val
+
+    Attributes:
+      type: Type of the read value
+      dest: Pointer to memory to write
+      src: source value
+      base: If not None, the object from which we are reading memory.
+            It's used to avoid the target object from being freed via
+            reference counting. If the target is not in reference counted
+            memory, or we know that the target won't be freed, it can be
+            None.
+    """
+    error_kind = ERR_NEVER
+
+    def __init__(self,
+                 type: RType,
+                 dest: Register,
+                 src: Value,
+                 base: Optional[Value],
+                 line: int = -1) -> None:
+        super().__init__(line)
+        self.type = type
+        self.src = src
+        self.dest = dest
+        self.base = base
+
+    def sources(self) -> List[Value]:
+        if self.base:
+            return [self.src, self.base]
```

Comment:
I think that `self.dest` should also be a source, since it's a value we access here.